pages/enterprise/cloud_forwarder.rst

```diff
@@ -0,0 +1,420 @@
+        
+Additionally, you can get deeper insights into Forwarder messaging and node health. Click the Details button on your 
+node, to get information about your message cache (buffers). One metric to pay attention to is the number of messages 
+in the journal. The journal is the on-disk persistent storage location that stores Forwarder messages. So, if the rest 
+of Graylog malfunctions, we can still keep all the messages we still have a place to put them.
+        
+Prometheus Metrics Exports
+--------------------------
+
+The Forwarder, alone, has no interface for insight into the internal operations. To that end, you must configure a 
+local Prometheus container; this becomes the interface for Forwarder metrics. These are similar to the traditional 
+Graylog Server metrics but instead are exported to Prometheus. The response format is the standard Prometheus HTTP 
+export format.
+        
+To start this process:
+
+#. Download and start `Prometheus <https://prometheus.io/docs/prometheus/latest/getting_started/#downloading-and-running-prometheus>`__.
+#. Install `Docker <https://docs.docker.com/get-docker/>`__ on your machine.
+#. Create a Prometheus Dockerfile, e.g. ``touch /tmp/prometheus.yml``:
+    
+    .. code-block:: yaml
+        
+        $ global:
+          scrape_interval: 15s
+          scrape_timeout: 10s
+          evaluation_interval: 15s
+        alerting:
+          alertmanagers:
+          - static_configs:
+            - targets: []
+            scheme: http
+            timeout: 10s
+            api_version: v1
+        scrape_configs:
+        - job_name: prometheus
+          honor_timestamps: true
+          scrape_interval: 15s
+          scrape_timeout: 10s
+          metrics_path: /api/metrics/prometheus
+          scheme: http
+          static_configs:
+          - targets:
+            - host.docker.internal:9001
+      
+#. Run this Docker command to start the container:
+
+    .. code-block:: bash
+
+            docker run \
+              -p 9090:9090 \
+              -v /tmp/prometheus.yml:/etc/prometheus/prometheus.yml \
+              prom/prometheus
+        
+*****************
+Resiliency Models
+*****************
+        
+When you think about scaling your deployment -- that is, add more Forwarders -- you must incorporate tools, procedures, 
+and policies that let you continue operating in the case of a major outage – widespread, long-lasting, destructive, or 
+all three. If all the above pose a threat to your Forwarder consider both message recovery and load balancing. 
+        
+Message Recovery
+================
+        
+The Cloud Forwarder’s disk journal is capable of caching data in case of a network outage. From there, they are read 
+and sent to Graylog Cloud. 
+        
+As mentioned in the `Output Framework chapter <https://docs.graylog.org/en/4.0/pages/integrations/output_framework.html?highlight=Journal#on-disk-journal>`__, if the internet is unavailable, 
```

Comment:
Same as link above. I think it's better to use [RST links](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-a-custom-anchor).